.github/workflows/pullrequest.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           export PR_REF=$GITHUB_SHA
           export SHELL=/usr/bin/bash
           export WORKFLOW_ID=pullrequest
-          source ./scripts/ci/00_prepare-base-tags.sh
+          source ./scripts/ci/00_prepare-base-tags.sh $(git merge-base HEAD $GITHUB_BASE_REF)
```

Comment:
What would this accomplish? Seems like a drastic change which might affect the CI pipeline?

Reply:
Nvm, I did this. This is a fix for when two branches with no built containers are merged. In that case we had a hard failure but this fixes that